### ISSUE TITLE
Easier introduction to format

### ISSUE THREAD

**Issue by ShadowManu** [json-api/json-api#923]
Hi. I'm working on a API service and my team found out this API spec. I personally liked the concept behind the project and we are going to follow the format. However, a issue we have at this is introducing the [format](http://jsonapi.org/format/) to the developers (including myself) which, let's be honest, it's way too long to get a fast grasp on.

Given this, wouldn't something like the graph and right table on the [JSON spec](http://www.json.org/) be beneficial to this project too? I could help on this, but I also want to know what you guys think of it.

**Comment by steveklabnik** [comment]
> it's way too long to get a fast grasp on.

Yup! We've wanted to build more introductory material, there just hasn't been time.

**Comment by ShadowManu** [comment]
Should I do a PR? and if I do, where should this introductory material should be located on the repo?

**Comment by tkellen** [comment]
Yes please! You can put it anywhere that we can see it--we can iterate on where exactly it winds up on the site once we have something to work with.

**Comment by jelhan** [comment]
Related #639
